src/SnackbarContainer.tsx

```diff
@@ -4,37 +4,71 @@ import { makeStyles } from '@material-ui/core/styles';
 import { SNACKBAR_INDENTS } from './utils/constants';
 import { SnackbarProviderProps } from '.';
 
-const useStyle = makeStyles(theme => ({
+const collapse = {
+    container: '& > .MuiCollapse-container',
+    wrapper: '& > .MuiCollapse-container > .MuiCollapse-wrapper',
```

Comment:
Oh, I guess the problem is these two selectors. Those selectors do not appear in our DOM structure so `pointer-events: all` does not apply.